lib/app-tray.js

```diff
@@ -45,10 +42,20 @@ exports.init = function (gme) {
     type: 'separator'
   }));
   trayMenu.append(new MenuItem({
+    label: 'Show / Hide Google Music Electron',
+    click: gme.onTrayClick
+  }));
```

Comment:
Clarity. Grouping two similar actions (hiding & quitting), and eliminating a separator. I also feel it makes the menu easier to scan.